module-4/backend/week_one.md

```diff
@@ -5,17 +5,30 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+The most useful thing I learned over intermission was how to separate functions and then call them from each other. 
 2. What are some tools to help debug JavaScript code?
+Debugger, alert, console.log, pryjs. 
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha, Chai
 4. What is the syntax for invoking a function?
+  functionName(param)
 5. What is CORS?
```

Comment:
Questions 5 & 6 are outdated questions from an old version of the repo. Please pull the latest version for next week